test/lit/passes/optimize-for-js.wast

```diff
@@ -80,4 +84,693 @@
    (i64.const 1)
   )
  )
+
+ ;; Unsigned div by const
+
+ ;; CHECK:      (func $div-unsigned-by0-i64 (param $x i64) (result i64)
+ ;; CHECK-NEXT:  (i64.const 0)
+ ;; CHECK-NEXT: )
+ (func $div-unsigned-by0-i64 (param $x i64) (result i64)
+  (i64.div_u
+   (local.get $x)
+   (i64.const 0)
+  )
```

Comment:
This doesn't make sense to me. Division by zero should trap, so why is it turned into 0?